From 9f687a409e5721d274a8bea7879cf1e7a9f9bbda Mon Sep 17 00:00:00 2001
From: Neal Gompa <ngompa13@gmail.com>
Date: Sun, 4 Oct 2020 15:14:39 -0400
Subject: [PATCH 1001/1002] meson: Switch to using external wlroots

wlroots is available in the distribution, so let's use it.
---
 meson.build | 6 ++----
 1 file changed, 2 insertions(+), 4 deletions(-)

diff --git a/meson.build b/meson.build
index d4fadab..c73f299 100644
--- a/meson.build
+++ b/meson.build
@@ -45,9 +45,7 @@ thread_dep = dependency('threads')
 cap_dep = cc.find_library('cap')
 sdl_dep = dependency('SDL2')
 
-wlroots_proj = subproject('wlroots', default_options:
-  ['default_library=static', 'examples=false'])
-wlroots_static_dep = wlroots_proj.get_variable('wlroots')
+wlroots_dep = dependency('wlroots', version: '>=0.11.0')
 
 shadercompiler = find_program('glslangValidator')
 
@@ -77,7 +75,7 @@ executable(
     dependencies : [
         dep_x11, dep_xdamage, dep_xcomposite, dep_xrender, dep_xext,
         dep_xxf86vm, pixman_dep, drm_dep, wayland_server, wayland_protos,
-        libinput, xkbcommon, math, thread_dep, sdl_dep, wlroots_static_dep,
+        libinput, xkbcommon, math, thread_dep, sdl_dep, wlroots_dep,
         vulkan_dep, libftoff_dep, dep_xtst, cap_dep
     ],
     install: true,
-- 
2.26.2

